beets/util/__init__.py

```diff
@@ -756,6 +756,8 @@ def interactive_open(target, command=None):
     else:
         base_cmd = open_anything()
         command = [base_cmd, base_cmd]
-
-    command.append(target)
+    if multiple_targets:
+        command = command + multiple_targets
+    else:
+        command.append(target)
```

Comment:
For future sanity, I think we should just bite the bullet and do the refactoring. That is, there would be only a single way to invoke the function, either with a list-of-strings argument (which can, after all, contain only a single string) or with `*args`. In either case, we will break backwards-compatibility, but there aren't that many call sites that need fixing. Sound OK?